packages/asc-ui/src/components/Accordion/Accordion.tsx

```diff
@@ -52,7 +62,9 @@ const Accordion: React.FC<
         onClick={handleClick}
         {...otherProps}
       >
-        <AccordionButtonContent>{title}</AccordionButtonContent>
+        <AccordionButtonContent singleLine={singleLine}>
```

Comment:
Can we rename singleLine to something more explaning, like wrapContent or withEllipsis?

Reply:
tbh, I think wrapContent or withEllipsis are not really self-explaining prop names... perhaps noMultiline?